src/eventSchemas/commands/searchSchemas.ts

```diff
@@ -173,7 +177,7 @@ export function createMessageReceivedFunc({
                 return
 
             case 'searchSchemas':
-                recordSchemaSearchTelemetry(telemetryService, message.command)
+                recordSchemasSearch({ result: 'Succeeded' })
```

Comment:
Please add a TODO based on @bryceitoc9 comment about metrics when the search fails

Reply:
Added above, why did you comment it on this line?